COORDINATE_SYSTEM2/COORDINATE_SYSTEM2/Point.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace COORDINATE_SYSTEM2
+{
+    internal class Point
+    {
+        private int x;
+        private int y;
+        public Point(int X, int Y)
+        {
+            this.x = X;
+            this.y = Y;
+        }
```

Comment:
Please add space after all "}"

Reply:
what do you mean?

Reply:
hmm, you mean new line?

Reply:
if so check my comment above https://github.com/fawkes4/csharp_FromZeroToHero/pull/2#discussion_r900407187

Reply:
done